### PR TITLE
cmd/juju/sla: Routine cleanup, renamed internal command type and removed TODOs that were DONE

### PR DESCRIPTION
**Pull request by mattyw**
## Description of change

Some TODOs had been left in the code, despite them already being done. Also renamed an internal type for the sake of neatness
## QA steps

None Required. Internal code cleanup only
## Documentation changes

N/A

## Bug reference

N/A

### CODE REVIEW

File: cmd/juju/romulus/sla/sla.go [hunk 1]
```diff
@@ -48,40 +48,38 @@ var modelId = func(conn api.Connection) string {
 	return tag.Id()
 }
 
-// TODO (mattyw) See juju/cmd/juju/storage/show.go for a better way of doing this.
-// TODO (mattyw) This should be fixed before this lands in master.
 // NewSLACommand returns a new command that is used to set sla credentials for a
```
Comment (alesstimec) [inline cmd/juju/romulus/sla/sla.go]: s/sla credentials/SLA credentials/

File: cmd/juju/romulus/sla/sla.go [hunk 2]
```diff
@@ -48,40 +48,38 @@ var modelId = func(conn api.Connection) string {
...
 	c.ModelCommandBase.SetFlags(f)
 	f.StringVar(&c.Budget, "budget", "", "the maximum spend for the model")
 }
 
 // Info implements cmd.Command.
-func (c *supportCommand) Info() *cmd.Info {
+func (c *slaCommand) Info() *cmd.Info {
 	return &cmd.Info{
 		Name:    "sla",
 		Aliases: []string{"support"},
```
Comment (alesstimec) [inline cmd/juju/romulus/sla/sla.go]: support?

Comment (alesstimec) [inline cmd/juju/romulus/sla/sla.go]: and Purpose: "Set the SLA level for a model."

**Comment by mattyw** [comment]
$$merge$$

**Comment by mattyw** [comment]
$$merge$$

**Comment by mattyw** [comment]
$$merge$$

**Pull request merged**